lib/logstash/inputs/file.rb

```diff
@@ -120,6 +121,10 @@ def register
                    :sincedb_path => @sincedb_path, :path => @path)
     end
 
+    if File.directory?(@sincedb_path)
+      raise ArgumentError.new("sincedb_path must be a file path, not a directory. Received: \"#{@sincedb_path}\"")
```

Comment:
"The \"sincedb_path\" argument must point to a file, received a directory: \"#{@sincedb_path}\""

What about this?